meta-lmp-base/recipes-containers/runc/runc-opencontainers_git.bb

```diff
@@ -1,10 +0,0 @@
-include recipes-containers/runc/runc.inc
-
-SRCREV = "51d5e94601ceffbbd85688df1c928ecccbfa4685"
```

Comment:
Fine to drop runc-opencontainers and containerd-opencontainers here, but my goal when dealing with these recipes in lmp was to avoid using rcs and beta releases from upstream, so just to keep in mind as we do the remaining updates. These components are critical to LmP, so better to use the released / stable versions instead (and we also need to validate if docker itself was tested and released with the same base versions here).

Reply:
I believe that in the LTS scarthgap branch of meta-virt we will only have stable versions.
However, for some time to come we will probably need to update these same stable. Maybe when we go back to following what's in the master again, there may be rcs and beta releases.
but agree that it is an important point to keep in mind.